server/index.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const db = require('../database/index.js');
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(__dirname + '/../dist/static/js/bundle.js'));
+
+const PORT = 3101;
+
+app.get('/tiersIncluded', (req, res) => {
+  const dataString = { bundleId: 1 };
+  return db.tiersRequest(dataString)
+  .then(tiersData => {
+    if (tiersData) {
+      return tiersData;
+    } else {
+      res.status(404).send('no such bundleId');
+    }
+  })
+  .then(tiersData => {
+    if (tiersData) {
+      res.send(JSON.stringify(tiersData));
```

Comment:
I do not think you need a conditional. You can just send it and if there is an error, I think your catch should automatically fire instead.